src/Python/qsharp-core/qsharp/ipython_magic.py

```diff
@@ -24,7 +24,7 @@ def qsharp(magic_args, cell, local_ns=None):
         callables = qs.compile(cell)
         if isinstance(callables, qs.QSharpCallable):
             local_ns[callables._name] = callables
-        else:
+        elif callables:
```

Comment:
This is slight change in behavior. 

Before, if `callables` was `None` an error would probably be raised when trying to iterate over it. Now, nothing would happen if `callables` is `None`. The user might not know what is going command silently does nothing.

Reply:
As discussed, this was actually a bug, and it does not change the behavior of the `%%qsharp` magic when compiling Q# code inline a Python notebook. But it fixes a `null reference` bug when calling another IQ# magic command that does not return a callable like
```
%%qsharp
%version
```